builds/fabric8-che/pom.xml

```diff
@@ -10,7 +10,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>@che.version@</version>
```

Comment:
OK, so what is the risk or complexity ? Just one line to change in your existing PR, or am I missing something ? 

Anyway, any idea about when you should be able to merge #89 ?
It would be useful for developers to have this PR merged while working on the remaining subtasks of [CHE-229](https://issues.jboss.org/browse/CHE-229) in the meantime.